test/kubemark/common.sh

```diff
@@ -49,8 +49,13 @@ export RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 # arguments:
 # $@: all stuff that goes after 'gcloud compute '
 function run-gcloud-compute-with-retries {
+  echo "" > /tmp/gcloud_retries
   for attempt in $(seq 1 ${RETRIES}); do
-    if ! gcloud compute $@; then
```

Comment:
That's a good point. Solved it simply by checking if the 'already exists' error comes up right in the first attempt (which means previous run didn't stop properly).